app-shells/squirrelsh/files/squirrelsh-1.2.7-gcc6.patch

```diff
@@ -0,0 +1,15 @@
+--- a/shell/base.cpp
++++ b/shell/base.cpp
+@@ -14,10 +14,11 @@
+ // You should have received a copy of the GNU General Public License
+ // along with this program. If not, see <http://www.gnu.org/licenses/>.
+ 
+-#include "common.h"
+ #include <string.h>
+ #include <string>
+ 
++#include "common.h"
```

Comment:
> from your report, would it be possible to fix this properly once and for all?
@SoapGentoo Upstream did the work and fixed this for good..  Pushed a new patch based on https://sourceforge.net/p/squirrelsh/git/ci/13280cdf34d96216b4d10b9ea7656a42a967fc2b/